interfaces/repo_test.go

```diff
@@ -1225,12 +1225,32 @@ func (s *RepositorySuite) TestConnectedFindsCoreSnap(c *C) {
 	}
 	c.Assert(s.testRepo.AddPlug(s.plug), IsNil)
 	c.Assert(s.testRepo.AddSlot(slot), IsNil)
-	_, err := s.testRepo.Connect(*NewConnRef(s.plug, slot), nil, nil, nil)
+	_, err := s.testRepo.Connect(NewConnRef(s.plug, slot), nil, nil, nil)
 	c.Assert(err, IsNil)
 
 	conns, err := s.testRepo.Connected("", s.slot.Name)
 	c.Assert(err, IsNil)
-	c.Check(conns, DeepEquals, []ConnRef{*NewConnRef(s.plug, slot)})
+	c.Check(conns, DeepEquals, []*ConnRef{NewConnRef(s.plug, slot)})
+}
+
+// Connected finds connections when asked from plug or from slot side
+func (s *RepositorySuite) TestConnections(c *C) {
```

Comment:
This looks ok, is it related to the change or just an addition from other PRs?

Reply:
Nah, it's merging mistake again, it's the function from below, only without all the changes I made. Thanks!